numpy/core/multiarray.py

```diff
@@ -251,6 +251,21 @@ def concatenate(arrays, axis=None, out=None, *, dtype=None, casting=None):
     return arrays
 
 
+def concat(arrays, axis=None, out=None, *, dtype=None, casting=None):
+    """
+    concat((a1, a2, ...), axis=0, out=None, dtype=None, casting="same_kind")
+
+    Join a sequence of arrays along an existing axis.
+
+    `concat` is an alias of `~numpy.concatenate`.
+
+    See Also
+    --------
+    concatenate : alias of this function 
+    """
+    return concatenate(arrays, axis=None, out=None, dtype=None, casting=None)
```

Comment:
```suggestion
    return concatenate(arrays, axis=axis, out=out, dtype=dtype, casting=casting)
```
Okay, this should fix the test failures. You were hardcoding the alias to ignore user input and set all keyword arguments to `None`, which caused incorrect output.